MovieApp/MovieApp/App/LocalDataSource/MovieLocalDataSource.swift

```diff
@@ -0,0 +1,37 @@
+import Combine
+import Foundation
+import RealmSwift
+
+class MovieLocalDataSource: MovieLocalDataSourceProtocol {
+
+    static let shared: MovieLocalDataSourceProtocol = MovieLocalDataSource()
+
+    func getMoviesPublisher(for category: CategoriesDataSource) -> AnyPublisher<[MovieDataModel], Never> {
+        guard let realm = try? Realm() else { return .empty() }
+
+        let movies = realm
+            .objects(MovieRealmDataModel.self)
+            .filter("category == %@", category.rawValue)
+            .map { MovieDataModel(fromModel: $0) }
+
+        return Just(Array(movies))
+            .eraseToAnyPublisher()
+    }
+
+    func saveLocal(array: [MovieDataModel], category: CategoriesDataSource) {
```

Comment:
It can be only `save`. The class is called `localDataSource` so the name indicates it's something local.